src/main/java/com/conveyal/datatools/manager/jobs/MonitorServerStatusJob.java

```diff
@@ -205,10 +210,10 @@ private void failJob(String message) {
         status.fail(String.format("%s Check logs at: %s", message, getUserDataLogS3Path()));
     }
 
-    /** Determine if job has passed time limit for its run time. */
-    private boolean jobHasTimedOut() {
+    /** Determine if a specific task has passed time limit for its run time. */
+    private boolean taskHasTimedOut(long startTime, long maxRunTime) {
```

Comment:
Please change `maxRunTime` and `maxGraphLoadWaitTime` so that the variable names include `Millis` so that it's easy to identify that they should contain a time value in milliseconds.